tests/system/providers/amazon/aws/example_http_to_s3.py

```diff
@@ -0,0 +1,83 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Really nice job on the system test! Thanks for following the guidance

Reply:
I have seen there is a [dashboard ](https://aws-mwaa.github.io/#/open-source/system-tests/dashboard.html) that runs these example dags.  How will this work for http endpoint? I'm wondering if I need to add something else.

Reply:
Glad to see you are using the dashboard :) To make it run, we (AWS team) needs to add a configuration on our side. Nothing you can do but once this PR is merged we'll definitely do that. Once done, your system test will show up on the dashboard :)